apps/meteor/client/views/omnichannel/contactHistory/ContactHistoryItem.tsx

```diff
@@ -21,8 +22,9 @@ type ContactHistoryItemProps = {
 
 function ContactHistoryItem({ history, setChatId, ...props }: ContactHistoryItemProps): ReactElement {
 	const t = useTranslation();
-	const formatDate = useTimeAgo();
+	const formatDate = useFormatDateAndTime();
 	const username = history.servedBy?.username;
+	const hasClosingMessage = !!history.closingMessage?.msg;
```

Comment:
how about str.trim() ?

Reply:
`     ` in case of space messages, right?